src/openapi/transport/batch.js

```diff
@@ -222,6 +224,16 @@ TransportBatch.prototype.addToQueue = function(item) {
     }
 };
 
+/**
+ * @private
+ * @param item
+ */
+TransportBatch.prototype.shouldQueue = function(item) {
+    const serviceOptions = this.services[item.servicePath] || {};

```

Comment:
optional chaining instead of creating an empty object?
if this uses preset-env it might require a babel upgrade though :S

Reply:
Yes it would upgrading various things including babel and eslint

Reply:
I think that's best not mixed in with this PR

Reply:
yes thats ok